tests/test_sync.py

```diff
@@ -648,11 +648,11 @@ def test_selection(self):
         with pytest.raises(FileSyncConflict):
             self.project_a.sync(self.project_b, selection=self.project_b.find_jobs({'a': 0}))
         with pytest.raises(FileSyncConflict):
-            self.project_a.sync(self.project_b, selection=self.project_a.find_job_ids({'a': 0}))
+            self.project_a.sync(self.project_b, selection=self.project_a.find_jobs({'a': 0}))
         with pytest.raises(FileSyncConflict):
-            self.project_a.sync(self.project_b, selection=self.project_b.find_job_ids({'a': 0}))
+            self.project_a.sync(self.project_b, selection=self.project_b.find_jobs({'a': 0}))
         f = {'a': {'$ne': 0}}
         self.project_a.sync(self.project_b, selection=self.project_a.find_jobs(f))
         self.project_a.sync(self.project_b, selection=self.project_b.find_jobs(f))
-        self.project_a.sync(self.project_b, selection=self.project_a.find_job_ids(f))
-        self.project_a.sync(self.project_b, selection=self.project_b.find_job_ids(f))
+        self.project_a.sync(self.project_b, selection=self.project_a.find_jobs(f))
+        self.project_a.sync(self.project_b, selection=self.project_b.find_jobs(f))
```

Comment:
The goal of this is to test that passing either `Job` instances or job id strings works, so you'll want to do the conversion here. Otherwise the two modified lines are identical to the two lines above them.